.github/workflows/on-pullrequest-dockerfile.yml

```diff
@@ -29,7 +29,6 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: write
-      security-events: write
```

Comment:
I think this is required. @msilesgap please review and confirm one way or another.

Reply:
NVM, I see that this is no longer required, because we had to disable the sarif file upload (lines 83-88).